docs/src/examples/01-loading-and-saving.jl

```diff
@@ -39,15 +39,16 @@ download_model(
     "b4db506aeed0e434c1f5f1fdd35feda0dfe5d82badcfda0e9d1342335ab31116",
 )
 
-# (You do not need to fill in the hash values immediately; simply run the
-# function once and then copy the reported hash value into your script.)
+#md #!!! tip "Tip: How do I get the model hash?"
+#md #    You do not need to fill in the hash values immediately; simply run the
+#md #    function once, and then copy the reported hash value into your script.
 
 # ## Loading models
 
-# To load the flux-balance constrained models, COBREXA uses the
-# [AbstractFBCModels](https://github.com/COBREXA/AbstractFBCModels.jl)
-# framework to import various kinds of models including SBML, JSON and the
-# legacy Matlab-formatted "COBRA toolbox" models.
+# To load genome-scale metabolic models, COBREXA uses the
```

Comment:
so if my model is not genome scale (or is pangenome), this won't work?

Reply:
hmm can of worms :)